docker-run.sh

```diff
@@ -100,31 +100,31 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
-echo "Copying out traced data..."
+echo "copying traced data from '$container:/data/output.json' to '$OUTPUT_JSON'"
```

Comment:
Are you logging for copies into the container too?

Reply:
I should probably do that too thanks